### ISSUE TITLE
WCAG: Time-based Media

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#326]
# Summary

## Required Action
- BirkbeckCTP/janeway#3935

## Context

There is limited video on the site, and no expectation of live video.  Where video is used, they are embedded from YouTube and have the options of auto-generated captions (as provided by YouTube).  These are not accurate.  

- include synchronised captions for videos
- include audio description tracks for videos
- include descriptive transcripts for videos
- where the video is of a presentation with slides (e.g powerpoint), if at all possible those slides should be included for download in an accessible format.  At the least, the text on the slides should be available in a transcript.

# Audit Results

## Automated Captions

Automatic captions are a starting point, but "Automatically-generated captions do not meet user needs or accessibility requirements, unless they are confirmed to be fully accurate." (W3.org)

Reviewing the videos on see https://www.openlibhums.org/site/webinars-archive/ it was quickly apparent that the automated captions are not accurate. As well as mistakes in the text itself, it is problematic that proper names are not recognised as such, and not capitalised making it difficult to follow the sense of the captions.  Apparently random terms such as 'knowledge' are capitalised while journal titles are not, and  'open library of humanities' and as 'olh' remain stubbornly lowercase.

## Audio Description

The videos do not offer audio description nor descriptive transcripts.  

where there is relevant visual content that is not presented in the audio, audio descriptions are required and additional descriptive transcripts are recommended.

Some of our video includes contents on slides - like a filmed powerpoint presentation.  It is unclear quite how videos of slides which contain text information should be handled under the WCAG, but if these were still images, then a text alternative would be required, and in general for video there should be audio description and preferably descriptive transcripts.  It seems desirable for the original presentation to be available for download (in accessible format) - and if this cannot be included, then all the text visible on those slides needs to be included in a descriptive transcript.
